src/utils/hooks/useTitle.ts

```diff
@@ -134,6 +141,10 @@ const MSG = defineMessages({
     id: 'utils.hooks.useTitle.fallbackTitle',
     defaultMessage: `Colony`,
   },
+  createExpenditure: {
+    id: 'utils.hooks.useTitle.createExpenditure',
+    defaultMessage: `Create Expenditure | Colony`,
```

Comment:
I missed this last time, can we please add ` - {colonyName}` here. As Creating an expenditure is in context of a colony.